src/Bridges/Bridges.jl

```diff
@@ -83,4 +83,6 @@ include("soctopsdbridge.jl")
 const SOCtoPSD{T, OT<:MOI.ModelLike} = SingleBridgeOptimizer{SOCtoPSDBridge{T}, OT}
 const RSOCtoPSD{T, OT<:MOI.ModelLike} = SingleBridgeOptimizer{RSOCtoPSDBridge{T}, OT}
 
+include("indicatorbridge.jl")
```

Comment:
It also needs to be added in `full_bridge_optimizer`

Reply:
what do you mean?

Reply:
ah ok got it, added in last commit